repos/kaleido/cc/kaleido.cc

```diff
@@ -560,6 +560,8 @@ void OnHeadlessBrowserStarted(headless::HeadlessBrowser* browser) {
         if (tagUrl.is_valid()) {
             // Value is a url, use a src of script tag
             htmlStringStream << "<script type=\"text/javascript\" src=\"" << tagValue << "\"></script>";
+        } else if  (tagValue.find("import") != std::string::npos && tagValue.find("+esm") != std::string::npos) {
```

Comment:
I think semantically it would make sense to have this check within the else below, not a separate else if.

Maybe it would also be helpful to extract this check to isScriptAModule() class method, or maybe even place it in utils.h, it would be easier for readability that way